package.json

```diff
@@ -53,7 +54,7 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "es-check": "^2.0.2",
     "eslint-config-edx": "^4.0.3",
-    "fetch-mock": "^6.3.0",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
You had another [PR](https://github.com/edx/front-end-cookie-cutter-application/pull/155/files) that changed to use MiniCssExtractPlugin instead. Should we remove this from the `package.json`?

Reply:
Thanks, missed that in the rebase!